byron/chain/executable-spec/src/Cardano/Spec/Chain/STS/Rule/Chain.hs

```diff
@@ -189,7 +189,7 @@ instance HasTrace CHAIN where
       -- current slot to a sufficiently large value.
       gCurrentSlot = Slot <$> Gen.integral (Range.constant 32768 2147483648)
 
-  sigGen = sigGenChain GenDelegation GenUTxO
+  sigGen _ = sigGenChain GenDelegation GenUTxO Nothing
```

Comment:
Am I correct in understanding that the occurrences of `sigGen _ = <...>` reference generators which do not currently implement the `PredicateFailure`-targeting functionality, and that we'll implement that functionality next, by modifying those generators? If so, looks good 👍

Reply:
Yes, that'd be the idea. Furthermore, I think we might need to define a `Invalid.Trace` data structure so that failures are incorporated and make this the default return type of `trace`. But I'd like to discuss this in detail with you and @nc6.